src/android/CodePushPackageManager.java

```diff
@@ -95,4 +95,16 @@ public InstallOptions getPendingInstall() {
     public void clearPendingInstall() {
         this.codePushPreferences.clearPendingInstall();
     }
+
+    public void markUnconfirmedInstall(boolean unconfirmed) {
```

Comment:
I changed the implementation to have two methods: markUnconfirmedInstall() and clearUnconfirmedInstall(), both not taking parameters.